dwave_networkx/generators/chimera.py

```diff
@@ -35,7 +35,18 @@
            ]
 
 
-def chimera_graph(m, n=None, t=None, create_using=None, node_list=None, edge_list=None, data=True, coordinates=False):
+def _add_compatible_edges(G, edge_list):
+    # Slow when edge_list is large, but clear (non-defaulted behaviour, so fine):
+    if edge_list is not None:
+        if not all([G.has_edge(*e) for e in edge_list]):
```

Comment:
the square brackets of the list comprehension are unnecessary, and trigger a memory allocation we don't need

`all(G.has_edge(*e) for e in edge_list)`